.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+This tracker is for bug reports only.
+
+
+Before opening an issue, please make sure you've checked the following:
+
+- For support requests, please use Stack Overflow (stackoverflow.com) or Gitter (see the README).
+
+- If the bug is in a plugin, open an issue on the plugin repository, not the gulp repository.
+
+- If you're getting a deprecated module warning, don't worry about it: we're aware of it and it's not an issue. To make it go away, update to Gulp 4.0.
```

Comment:
This should probably link the an issue with an explanation. Is there one for minimatch, or just for lodash?